bot/tests/processors/test_greeting.py

```diff
@@ -1,10 +1,11 @@
-from bot.processors import Bot, GreetingBotModule, Event
+from penny_university.bot.processors.base import Event
+from penny_university.bot.processors.greeting import GreetingBotModule
 
 
 def test_greeting(mocker):
     slack = mocker.Mock()
     greeter = GreetingBotModule(slack)
-    bot = Bot(event_processors=[greeter])
```

Comment:
we were testing both the `Bot` and `GreetingBotModule` so I removed the bot part so we could test them separately